test/e2e_node/image_list.go

```diff
@@ -139,6 +141,10 @@ func (rp *remotePuller) Pull(image string) ([]byte, error) {
 	return nil, err
 }
 
+func (rp *remotePuller) Remove(image string) error {
+	return rp.imageService.RemoveImage(context.Background(), &runtimeapi.ImageSpec{Image: image})
```

Comment:
Would it make sense to pass a test case context instead of creating a separate one with context.Background() ?

Reply:
Updated.